### ISSUE TITLE
When hovering over scalar data the results are out of screen 

### ISSUE THREAD

**Issue by forReason** [tensorflow/tensorboard#3175]
## Environment information (required)

Please run `diagnose_tensorboard.py` (link below) in the same
environment from which you normally run TensorFlow/TensorBoard, and
paste the output here:

### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version d515ab103e2b1cfcea2b096187741a0eeb8822ef

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=6, micro=8, releaselevel='final', serial=0)
INFO: os.name: nt
INFO: os.uname(): N/A
INFO: sys.getwindowsversion(): sys.getwindowsversion(major=10, minor=0, build=14393, platform=2, service_pack='')

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==2.1.0
INFO: installed: tensorflow==2.1.0
INFO: installed: tensorflow-estimator==2.1.0

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.1.0'

--- check: tensorflow_python_version
2020-01-21 18:44:11.139289: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_101.dll'; dlerror: cudart64_101.dll not found
2020-01-21 18:44:11.143821: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
INFO: tensorflow.__version__: '2.1.0'
INFO: tensorflow.__git_version__: 'v2.1.0-rc2-17-ge5bf8de410'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'C:\\Program Files\\Python36\\Scripts\\tensorboard.exe\r\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'DRG-APP01.home'

--- check: stat_tensorboardinfo
INFO: directory: C:\Users\drgadmin\AppData\Local\Temp\1\.tensorboard-info
INFO: .tensorboard-info directory does not exist

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['C:\\Program Files\\Python36\\lib\\site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.9.0
astor==0.8.1
cachetools==4.0.0
certifi==2019.11.28
chardet==3.0.4
gast==0.2.2
google-auth==1.10.0
google-auth-oauthlib==0.4.1
google-pasta==0.1.8
grpcio==1.26.0
h5py==2.10.0
idna==2.8
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
Markdown==3.1.1
numpy==1.18.1
oauthlib==3.1.0
opt-einsum==3.1.0
pip==19.3.1
protobuf==3.11.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
requests==2.22.0
requests-oauthlib==1.3.0
rsa==4.0
scipy==1.4.1
setuptools==45.0.0
six==1.13.0
tensorboard==2.1.0
tensorflow==2.1.0
tensorflow-estimator==2.1.0
termcolor==1.1.0
urllib3==1.25.7
Werkzeug==0.16.0
wheel==0.33.6
wrapt==1.11.2

``````

</details>

### Next steps

No action items identified. Please copy ALL of the above output,
including the lines containing only backticks, into your GitHub issue
or comment. Be sure to redact any sensitive information.

For browser-related issues, please additionally specify:

  - Version 79.0.3945.130 (Offizieller Build) (64-Bit)
  - Screenshot, if it’s a visual issue:
![image](https://user-images.githubusercontent.com/12736950/73175524-b1ff5300-410a-11ea-9f47-34bd17ff3bf3.png)

![image](https://user-images.githubusercontent.com/12736950/73175247-035b1280-410a-11ea-9180-2d831456d716.png)

## Issue description

Expected behavior:
When hovering over scalar Data, a (scrollable) list is shown with the results of the different runs

Actual Behavior:
When hovering over scalar data (especially epoch_loss), a list with the runs is shown but it might be so far out of screen that not all records are shown.  For the lowest diagram (in my case epoch_loss) not even one run is shown fully because the overlay starts so deep out of screen. 

This especially comes to play when comparing more than 5-10 runs. The overlay is not scrollable, also the Webpage cant be scrolled down further.

**Comment by wchargin** [comment]
Hi @forReason! Thanks for the report. In some cases it can be helpful to
change the tooltip sorting method (in the top left of the options panel,
just above the smoothing slider) so that more relevant runs appear above
the fold: e.g., set it to “descending” if looking at accuracy or
“ascending” if looking at loss. This is often sufficient for a
workaround, but you’re certainly right that it would be nicer if the
tooltip were scrollable.

I’ll retain this as a feature request; we probably won’t get around to
addressing it soon, but we’d be happy to take a look at any PRs!

**Comment by forReason** [comment]
Hello @wchargin 

thank you for the Advice, the Workaround helps a lot. Although, it is a workaround. For my part, the Issue can be closed but I leave that decision to you.

I would love to commit to tensorflow / tensorboard but my programming language of choice is C# and I am already struggling to put together my 250 line training code in python.
But maybe in the future when I am more confident with python.

Best regards,

Julian

**Comment by wchargin** [comment]
Cool, glad to hear that the workaround is helpful. I’ll keep this open,
because you’re right that it’s just a workaround and this really should
be fixed.

> I would love to commit to tensorflow / tensorboard but my programming
> language of choice is C#

No problem at all—contributions are always appreciated but never
expected. :-)

**Comment by wchargin** [comment]
Duplicate of #1793
